### PR TITLE
getImageReverted method to see Sanger electropherogram in RC.

### PR DESCRIPTION
**Pull request by el1jaC**
Hi all,

I've developed a new method in ABITrace.java to see image of Sanger sequencing result in RC.

### CODE REVIEW

**Comment by el1jaC** [comment]
Hi @josemduarte ,
I think is possible using for example Java 8 optional parameters.
As soon as i can grab a minute I'll do that.

**Comment by el1jaC** [comment]
Sorry, I've committed branch version, I'll correct this in the afternoon.

File: biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/ABITrace.java [hunk 1]
```diff
@@ -212,9 +215,10 @@ public int getTraceLength() {
      *
      * @param imageHeight - desired height of the image in pixels.
      * @param widthScale - how many horizontal pixels to use to represent a single x-coordinate (try 2).
+     * @param isForward - if it has "true" value, the image is reverted.
      * @return BufferedImage image
      */
-    public BufferedImage getImage(int imageHeight, int widthScale) {
```
Comment (sbliven) [inline biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/ABITrace.java]: Please overload the method to include the old signature.

File: biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/ABITrace.java [hunk 2]
```diff
@@ -223,8 +227,12 @@ public BufferedImage getImage(int imageHeight, int widthScale) {
         Color tcolor = Color.red;
         Color ncolor = Color.pink;
         double scale = calculateScale(imageHeight);
-        int[] bc = baseCalls;
+        int[] bc = isForward ? baseCalls : getSequenceReverted(baseCalls);
         char[] seq = sequence.toCharArray();
+        int[] AS = isForward ? A : getSequenceReverted(A);
+        int[] CS = isForward ? C : getSequenceReverted(C);
+        int[] GS = isForward ? G : getSequenceReverted(G);
+        int[] TS = isForward ? T:  getSequenceReverted(T);
```
Comment (sbliven) [inline biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/ABITrace.java]: Maybe I'm misunderstanding, but shouldn't you perform the complementation step here?

E.g. `int[] AS = isForward ? A : getSequenceReverted(T);`

File: biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/ABITrace.java [hunk 3]
```diff
@@ -34,6 +34,7 @@
 import java.awt.image.BufferedImage;
 import java.net.URL;
 import java.io.InputStream;
+import java.util.Optional;
```
Comment (sbliven) [inline biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/ABITrace.java]: This isn't used.

**Pull request closed without merging**